core/amber/src/main/scala/edu/uci/ics/amber/engine/common/storage/HDFSRecordStorage.scala

```diff
@@ -7,30 +7,27 @@ import org.apache.hadoop.fs.{FileSystem, Path}
 
 import java.net.URI
 
-import java.net.URI
-import org.apache.hadoop.fs.{FileSystem, Path}
-import org.apache.hadoop.conf.Configuration
-
-class URIRecordStorage[T >: Null <: AnyRef](logFolderURI: URI)
+class HDFSRecordStorage[T >: Null <: AnyRef](hdfsLogFolderURI: URI)
     extends SequentialRecordStorage[T]
     with LazyLogging {
+
+  // only support hdfs uris
+  assert(hdfsLogFolderURI.getScheme.toLowerCase == "hdfs")
+
   private var fileSystem: FileSystem = _
   private val fsConf = new Configuration()
   // configuration for HDFS
   fsConf.set("dfs.client.block.write.replace-datanode-on-failure.enable", "false")
-  // configuration for disabling SUCCESS files
-  fsConf.set("mapreduce.fileoutputcommitter.marksuccessfuljobs", "false")
```

Comment:
is this removal intentional?

Reply:
Yes. This prevents Hadoop from writing additional files on the local file system. Since we use VFS for the local file system, this flag is no longer needed.